sdks/python/apache_beam/pipeline.py

```diff
@@ -123,20 +123,16 @@ class Pipeline(object):
   should be used to designate new names
   (e.g. ``input | "label" >> my_transform``).
   """
-
-  # TODO: BEAM-9001 - set environment ID in all transforms and allow runners to
-  # override.
   @classmethod
```

Comment:
Can we add an assert similar to PipelineValidator.java ?

Reply:
Where do you suggest the assert go?

The PipelineValidator does the assertion since it is used once the entire pipeline is constructed.

Reply:
Probably we can introduce a new visitor and update runners to use that similar to following ?
https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/dataflow/dataflow_runner.py#L557

But this is not a blocker.